Firebase/Storage/FIRStorageUploadTask.m

```diff
@@ -72,6 +72,13 @@ - (void)dealloc {
 - (void)enqueue {
   __weak FIRStorageUploadTask *weakSelf = self;
 
+  NSError *contentValidationError;
```

Comment:
Should this be move to the dispatchAsync block below to execute close to the actual upload?

Reply:
yeah, that makes sense